packages/react-core/src/components/TreeView/TreeView.tsx

```diff
@@ -59,7 +59,7 @@ export interface TreeViewProps {
   /** Flag indicating if all nodes in the tree view should have badges. */
   hasBadges?: boolean;
   /** Flag indicating if all nodes in the tree view should have checkboxes. */
-  hasChecks?: boolean;
+  hasCheckboxs?: boolean;
```

Comment:
`hasCheckboxs` instances just need to be updated to:

```suggestion
  hasCheckboxes?: boolean;
```